server/routes/api/auth.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcrypt');
+
+// User Model
+const User = require('../../models/user');
+
+// POST User Login
+router.post('/login', login);
+
+function login(req, res) {
+  const email = req.body.email;
+  const password = req.body.password;
+
+  User.authenticate(email, password, function(err, user) {
+    if (err) {
+      return res.json({ message: err.message });
```

Comment:
If you change the error structure in the model method, then we could just pass the `err` through here without putting it on a `message` property. That'll keep our error objects consistent.

Reply:
Updated for changes in the User model.